modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -28,7 +28,13 @@ export const FileTableComponent = (props: TTableProps) => {
 
   return (
     <div>
+
       <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between' }}>
+        {props.isCartPage &&
```

Comment:
Alternative solutions: when that happens, e.g. in a project entity page, we can just rename this variable to be `isCartOrProjectPage`. If another page needs it, we can rename it to `isCartOrProjectOr[OtherPage]Page`

Alternatively we can designate `isCartPage` to really mean "should show add to cart button" and either add that as a comment or stick it in our README